function/colcon_cd.sh

```diff
@@ -0,0 +1,79 @@
+# copied from colcon_cd/function/colcon_cd.sh
+
+colcon_cd() {
+  if [ $# = 0 ]; then
+    # change the working directory to the previously saved path
+    if [ "$_colcon_cd_root" = "" ]; then
+      echo "No previous path saved. Either call 'colcon_cd <pkgname>' from a" \
+        "directory where <pkgname> can be found or 'colcon_cd --set' to" \
+        "directly save the current working directory." 1>&2
+      return 1
+    fi
+    if [ "$_colcon_cd_root" != "$(pwd)" ]; then
```

Comment:
Just curious, what's wrong with changing directories even if it's the same as the current directory? Seems simpler to drop the if-condition.

Reply:
Then e.g. `cd -` wouldn't switch back anymore but be stuck in the current one.